spark/Dockerfile

```diff
@@ -1,41 +1,28 @@
 # syntax=docker/dockerfile:1
-FROM ubuntu:20.04
+FROM python:3.9-bullseye
```

Comment:
This way we don't have to uninstall/install Python. Bullseye is the same Debian version that ubuntu 20.04 is based on.

Reply:
Awesome, thanks @Fokko !